### PR TITLE
Jhalackova jeos slemicro

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Added jeos_firstboot

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/deployment_manual_installation.xml [hunk 1]
```diff
@@ -520,35 +520,47 @@ SLEMicro    cc36aae1
...
-   Patching</literal> extension that enables you to apply critical patches
+   &slema; currently offers the following extensions:
+</para>
+<variablelist>
+  <varlistentry>
+    <term><literal>SUSE Linux Enterprise Live Patching</literal></term>
+    <listitem>
+      <para>
+        extension that enables you to apply critical patches
    without rebooting your system. Bear in mind that you might need an
```
Comment (dariavladykina) [inline xml/deployment_manual_installation.xml:531]:
```suggestion
        An extension that lets you apply critical patches without
        rebooting your system. Bear in mind that you might need an
```

File: xml/deployment_manual_installation.xml [hunk 2]
```diff
@@ -520,35 +520,47 @@ SLEMicro    cc36aae1
...
    </para>
   </note>
- </sect1>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><literal>SUSE Package Hub</literal></term>
+    <listitem>
+      <para>
+        is a free module that provides access to community maintained packages. Packages in the Package Hub are approved by &suse; for use on &sls;, thus the packages might not be installable on &slema;.
```
Comment (dariavladykina) [inline xml/deployment_manual_installation.xml:553]:
```suggestion
        A free module that provides access to community-maintained packages. Packages in the Package Hub are approved by &suse; for use on &sls;, thus the packages might not be installable on &slema;.
```

File: xml/deployment_postinstall_slemicro.xml [hunk 3]
```diff
@@ -69,30 +69,38 @@
...
   </para>
-
-  <note>
-   <title><literal>SUSE Linux Enterprise Live Patching</literal> availability</title>
+<variablelist>
+    <varlistentry>
+        <term><literal>SUSE Linux Enterprise Live Patching</literal></term>
+        <listitem>
+            <para>
+                the extension is available only for the &x86; (except for the real-time kernel) and
```
Comment (dariavladykina) [inline xml/deployment_postinstall_slemicro.xml:79]:
```suggestion
               The extension is available only for the &x86; (except for the real-time kernel) and
```

File: xml/deployment_postinstall_slemicro.xml [hunk 4]
```diff
@@ -69,30 +69,38 @@
...
+    &zseries; architectures. Bear in mind that the
+   extension might require an additional subscription.
+            </para>
+        </listitem>
+    </varlistentry>
+    <varlistentry>
+        <term><literal>PackageHub</literal></term>
+        <listitem>
+            <para>
+                is a free module that provides access to community maintained packages. Packages in the Package Hub are approved by &suse; for use on &sls;, thus the packages might not be installable on &slema;.
```
Comment (dariavladykina) [inline xml/deployment_postinstall_slemicro.xml:89]:
```suggestion
                A free module that provides access to community-maintained packages. Packages in the Package Hub are approved by &suse; for use on &sls;, thus the packages might not be installable on &slema;.
```

File: xml/deployment_postinstall_slemicro.xml [hunk 5]
```diff
@@ -69,30 +69,38 @@
...
    your system to boot to the new snapshot and apply the changes.
   </para>
 
   <sect2 xml:id="sec-slemicro-live-patching">
-   <title>Activating <literal>SUSE Linux Enterprise Live Patching</literal></title>
+   <title>Activating extensions</title>
    <para>
-    To active the <literal>SUSE Linux Enterprise Live Patching</literal>
-    proceed as follows:
+    To active an extension proceed as follows:
```
Comment (dariavladykina) [inline xml/deployment_postinstall_slemicro.xml:103]:
```suggestion
    To activate an extension, proceed as follows:
```

File: xml/deployment_postinstall_slemicro.xml [hunk 6]
```diff
@@ -69,30 +69,38 @@
...
+        <listitem>
+            <para>
+                the extension is available only for the &x86; (except for the real-time kernel) and
+    &zseries; architectures. Bear in mind that the
+   extension might require an additional subscription.
+            </para>
+        </listitem>
+    </varlistentry>
+    <varlistentry>
+        <term><literal>PackageHub</literal></term>
```
Comment (dariavladykina) [inline xml/deployment_postinstall_slemicro.xml:86]:
```suggestion
        <term><literal>Package Hub</literal></term>
```

File: xml/deployment_postinstall_slemicro.xml [hunk 7]
```diff
@@ -69,30 +69,38 @@
...
    <para>
-    The <literal>SUSE Linux Enterprise Live Patching</literal> extension is
-    available only for the &x86; (except for the real-time kernel) and
-    &zseries; architectures.
-   </para>
-  </note>
-
-  <para>
    As the extension activation or deactivation is performed as a
    transactional-update and thus creates a new snapshot, you need to restart
```
Comment (dariavladykina) [inline xml/deployment_postinstall_slemicro.xml:96]: Do we need to mark transactional-update as a command?

File: xml/deployment_postinstall_slemicro.xml [hunk 8]
```diff
@@ -103,19 +111,23 @@
     </step>
     <step>
      <para>
-      The output provides you with a command regarding how to activate the live
-      patching extension. You can skip this step if you activated the extension
-      during the manual installation.
+      The output provides you with a command regarding how to activate the extensions, for example for the <literal>SUSE Linux Enterprise Live Patching</literal>:
```
Comment (dariavladykina) [inline xml/deployment_postinstall_slemicro.xml:114]:
```suggestion
      The output provides you with a command to activate the extensions, for example, for the <literal>SUSE Linux Enterprise Live Patching</literal>: 
```

File: xml/deployment_selfinstall_procedure.xml [hunk 9]
```diff
@@ -39,30 +39,32 @@
     </listitem>
     <listitem>
       <para>
-        a device that serves as a configuration medium. To prepare the configuration device, follow the steps in <xref linkend="pro-selfinstall-configuration-device"/>.
+        (optionally) a device that serves as a configuration medium. To prepare the configuration device, follow the steps in <xref linkend="pro-selfinstall-configuration-device"/>.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:42]:
```suggestion
        (optionally) a device that serves as a configuration medium. To prepare the configuration device, follow the steps in <xref linkend="pro-selfinstall-configuration-device"/>
```

File: xml/deployment_selfinstall_procedure.xml [hunk 10]
```diff
@@ -39,30 +39,32 @@
...
-        <para>
-          Bear in mind that the configuration device must be plugged in throughout the whole configuration process on the first boot. It is recommended to plug in the device before starting the installation process. However, if your firmware does not support having attached two or more USB disks on boot, you can exchange the USB device before you start the configuration process.
-        </para>
-      </note>
-    </listitem>
+      </listitem>
   </itemizedlist>
+  
+    <para>
+If you do not connect the configuration device during the installation, <emphasis>jeos-firstboot</emphasis> is started during the first boot. This tool enables you to perform a minimal configuration of &slema;. For the procedure, refer to <xref linkend="sec-jeos-firstboot-selfinstall"/>. If you need a more complex configuration, you need the configuration medium.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:48]:
```suggestion
If you do not connect the configuration device during the installation, <emphasis>jeos-firstboot</emphasis> is started during the first boot. This tool lets you perform a minimal configuration of &slema;. For the procedure, refer to <xref linkend="sec-jeos-firstboot-selfinstall"/>. If you need a more complex configuration, you need the configuration medium.
```

File: xml/deployment_selfinstall_procedure.xml [hunk 11]
```diff
@@ -39,30 +39,32 @@
...
+        <title>The configuration device must be plugged in during the first boot.</title>
+        <para>
+          Bear in mind that, if used, the configuration device must be plugged in throughout the whole configuration process on the first boot. It is recommended to plug in the device before starting the installation process. However, if your firmware does not support having attached two or more USB disks on boot, you can exchange the USB device before you start the configuration process.
+        </para>
+      </note>
+  <procedure xml:id="proc-selfinstall-installation">
     <title>&slema; installation</title>
+    <step>
+      <para>
+        If needed, prepare the configuration device as described in <xref linkend="pro-selfinstall-configuration-device"/>.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:65]:
```suggestion
        If needed, prepare the configuration device as described in <xref linkend="pro-selfinstall-configuration-device"/>
```

File: xml/deployment_selfinstall_procedure.xml [hunk 12]
```diff
@@ -85,24 +87,165 @@
     </step>
     <step>
       <para>
-        Start the configuration process by selecting <guimenu>SLE Micro</guimenu>. &slema; is configured according to the instructions provided on the configuration device.
+        Start the configuration process by selecting <guimenu>SLE Micro</guimenu>. &slema; is either configured according to the instructions provided on the configuration device or <emphasis>jeos-firstboot</emphasis> is started.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:90]:
```suggestion
        Start the configuration process by selecting <guimenu>SLE Micro</guimenu>. &slema; is either configured according to the instructions provided on the configuration device, or <emphasis>jeos-firstboot</emphasis> is started.
```

File: xml/deployment_selfinstall_procedure.xml [hunk 13]
```diff
@@ -85,24 +87,165 @@
...
+  </para>
+<procedure xml:id="pro-selfinstall-configuration-device">
+  &pro-preparing-configuration-device;
+</procedure>
+</section>
+
+<section xml:id="sec-jeos-firstboot-selfinstall">
+  <title>Minimal initial configuration</title>
+<para>
+  If you do not attach any configuration medium, <emphasis>jeos-firstboot</emphasis>  is started and you can perform minimal configuration of your system as follows:
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:125]:
```suggestion
  If you do not attach any configuration medium, <emphasis>jeos-firstboot</emphasis> is started, and you can perform a minimal configuration of your system as follows:
```

File: xml/deployment_selfinstall_procedure.xml [hunk 14]
```diff
@@ -85,24 +87,165 @@
...
+     <imagedata fileref="slemicro_jeos_keyboard.png" width="100&#37;"/>
+    </imageobject>
+    <textobject role="description"><phrase>jeos keyboard selection</phrase>
+    </textobject>
+        </mediaobject>
+      </informalfigure>
+      </step>
+      <step>
+        <para>
+          Read the licence agreement and accept it.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:165]:
```suggestion
          Read the license agreement and accept it.
```

File: xml/deployment_selfinstall_procedure.xml [hunk 15]
```diff
@@ -85,24 +87,165 @@
...
+     <imagedata fileref="slemicro_jeos_timezone.png" width="100&#37;"/>
+    </imageobject>
+    <textobject role="description"><phrase>jeos time zone selection</phrase>
+    </textobject>
+        </mediaobject>
+      </informalfigure>
+      </step>
+      <step>
+        <para>
+          Enter a password for the &rootuser;.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:199]:
```suggestion
          Enter a password for &rootuser;.
```

File: xml/deployment_selfinstall_procedure.xml [hunk 16]
```diff
@@ -39,30 +39,32 @@
...
-<para>
-  After you have prepared the configuration device, you can begin the installation process as described below.
+  To install &slema;, proceed as described in <xref linkend="proc-selfinstall-installation"/>.
   </para>
-
-  <procedure>
+<note>
+        <title>The configuration device must be plugged in during the first boot.</title>
+        <para>
+          Bear in mind that, if used, the configuration device must be plugged in throughout the whole configuration process on the first boot. It is recommended to plug in the device before starting the installation process. However, if your firmware does not support having attached two or more USB disks on boot, you can exchange the USB device before you start the configuration process.
```
Comment (dariavladykina) [inline xml/deployment_selfinstall_procedure.xml:58]:
```suggestion
          Bear in mind that, if used, the configuration device must be plugged in throughout the whole configuration process on the first boot. It is recommended to plug in the device before starting the installation process. However, if your firmware does not support having two or more USB disks attached on boot, you can exchange the USB device before starting the configuration process.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please consider some suggestions here. Thanks!

File: xml/deployment_postinstall_slemicro.xml (hunk 7 above)
Comment (lvicoun) [reply xml/deployment_postinstall_slemicro.xml:96]: Yes.

File: xml/deployment_postinstall_slemicro.xml [hunk 17]
```diff
@@ -69,30 +69,38 @@
...
    <para>
-    The <literal>SUSE Linux Enterprise Live Patching</literal> extension is
-    available only for the &x86; (except for the real-time kernel) and
-    &zseries; architectures.
-   </para>
-  </note>
-
-  <para>
    As the extension activation or deactivation is performed as a
    transactional-update and thus creates a new snapshot, you need to restart
```
Comment (lvicoun) [inline xml/deployment_postinstall_slemicro.xml:96]:
```suggestion
   <command>transactional-update</command> command and thus creates a new snapshot, you need to restart
```

**Pull request merged**